models/org_team.go

```diff
@@ -583,16 +575,19 @@ func removeTeamMember(ctx context.Context, team *organization.Team, userID int64
 		}
 	}
 
+	return removeInvaildOrgUser(ctx, e, userID, team.OrgID)
+}
+
+func removeInvaildOrgUser(ctx context.Context, sess db.Engine, userID, orgID int64) error {
```

Comment:
We just need ctx and use `db.GetEngine(ctx)`, so `sess db.Engine` could be removed.`

Reply:
Already fixed. 😄